### PR TITLE
Feature supervisor label truetype

### PR DESCRIPTION
**Pull request by DavidMansolino**
Linked with omichel/webots#4140

Link: https://www.cyberbotics.com/doc/reference/supervisor?version=feature-supervisor-label-truetype

### CODE REVIEW

**Comment by DavidMansolino** [comment]
:+1:

**Comment by omichel** [comment]
You didn't documented the default font value for C++/Java/Python.

File: reference/supervisor.md [hunk 1]
```diff
@@ -1034,19 +1048,19 @@ new robot with a specific controller:
 
 ```c
 #include <webots/robot.h>
-    include <webots/supervisor.h>
+include <webots/supervisor.h>
```
Comment (stefaniapedrazzi) [inline reference/supervisor.md]: the initial ``#`` is missing

**Comment by DavidMansolino** [comment]
:+1:

**Comment by omichel** [comment]
:+1:

File: reference/java-api.md [hunk 2]
```diff
@@ -728,6 +728,8 @@ The following tables describe the Java classes and their methods.
 | &nbsp;&nbsp; public [Node](#java_node) [getFromDef](supervisor.md#wb_supervisor_node_get_from_def)(String name);                                              |
 | &nbsp;&nbsp; public [Node](#java_node) [getFromId](supervisor.md#wb_supervisor_node_get_from_def)(int id);                                                    |
 | &nbsp;&nbsp; public void [setLabel](supervisor.md#wb_supervisor_set_label)(int id, String label, double xpos, double ypos,                                    |
+| &nbsp;&nbsp;&nbsp;&nbsp; double size, int color, double transparency, String font);                                                                           |
+| &nbsp;&nbsp; public void [setLabel](supervisor.md#wb_supervisor_set_label)(int id, String label, double xpos, double ypos,                                    |
```
Comment (stefaniapedrazzi) [inline reference/java-api.md]: is this ``setLabel`` function duplication correct?

Comment (DavidMansolino) [inline reference/java-api.md]: Yes, this is how swig handle default argument in Java, it creates 2 functions, 1 with the argument and another without.

Comment (stefaniapedrazzi) [inline reference/java-api.md]: Oh yes, in Java the default argument is not supported.
Thank you for the explanation.

**Pull request merged**